torch/lib/c10d/ProcessGroup.hpp

```diff
@@ -157,6 +160,12 @@ class ProcessGroup {
     return size_;
   }
 
+  // *************************************************************************
```

Comment:
shall we add this comment in the beginning of `ProcessGroup` class instead?

Reply:
moved